src/app/components/user-deposit-dialog/deposit-withdraw-form/deposit-withdraw-form.component.html

```diff
@@ -0,0 +1,41 @@
+<div
+    fxLayout="row"
+    fxLayoutAlign="center start"
+    [formGroup]="form"
+    (keyup.enter)="onEnter($event)"
```

Comment:
Is `keyup.enter` the best event for forms? Doesn´t it have a more generic submit which would streamline how forms submission is performed? just curious, as handling key events directly for forms seems quite spartane

Reply:
I could get it working without the enter event by appropriately using the form tag and setting `type="submit"` on the button. Thanks for the hint. I also improved this on other parts of the WebUI.